lib/assets/products.csv

```diff
@@ -1,28 +1,32 @@
 Atlas Knee Supporter,Spark-Seed-Spread,FACILITY USE,"The Atlas Knee Supporter was designed as a supportive device to immobilize Veterans and provide comfort during an otherwise challenging knee pain management procedures. It could, however, be used for any similar procedure where a Veteran needs to be secured and stable. The device comes in two sizes for different size legs and supports comfortably from the hip to the ankle.",Coming soon! (001),Coming soon!,Harbor Designs and Manufacturing,01-007-5243,This product was just licensed and is expected to ramp-up production in the fall. Shipments are expected to begin in 2024.,"""I really felt bad for the patient and knew there had to be a way to streamline the procedure while giving comfort to the Veteran. With the very first trial the patient loved it and wanted to take it home!! Throughout the prototyping phase I had such an adrenaline rush from any design changed needed. Since 2021, the need to reschedule patients is at ZERO, and knee procedure time has been reduced by 15 minutes. Previously, 20% of patients were rescheduled due to pain, inability to hold still, or needing sedation. The benefits of the supporter and the data I collected include decreased radiation exposure to patient and staff, rescheduling, use of anesthesia and procedure time. The meaning of VA Innovation to me is knowing that I have lots of support behind me. I have had many people cheering me on and giving great suggestions. VA Innovation is the way to go!!!!""
 
 Debra (DJ) Cole
 Radiologic Technologist
-VA Richmond Healthcare System"
+VA Richmond Healthcare System","Debra (DJ) Cole
+Radiologic Technologist"
 Cart Sweeper Attachment,Spark-Seed-Spread,FACILITY USE,"The Cart Sweeper Attachment is a simple device which attaches to the bottom of a housekeeping/janitorial cart. The device allows you to sweep the floors as the cart is pushed along its usual route throughout the hospital. This tool improves the cleanliness of the hospital environment by keeping floors clear of dust, dirt, and debris.",Coming soon! (002),Coming soon!,Harbor Designs and Manufacturing,01-007-5243,This product was just licensed and is expected to ramp-up production in the fall. Shipments are expected to begin in 2024.,"""We can work smarter, not harder. Developing this product opened me up to a large network and the depth of iNET support that VA offers when they bring an idea to the forefront. That was an eye-opener. It has allowed me to help others who have had similar innovations to steer them in the right direction.
 
 VA Innovation made me realize the VA is one of the biggest innovators out there in the nation's healthcare system! We are on the forefront of many new products in our industry!""
 
 Larry Hillson
 Chief, Environmental Management Service
-VA Sierra Nevada Healthcare System"
+VA Sierra Nevada Healthcare System","Larry Hillson
+Chief, Environmental Management Service"

```

Comment:
Can we update these so that the VA medical center is included with the person's role? So:a `VaEmployee` would have:
```
name: "Larry Hillson",
role: "Chief, Environmental Management Service, VA Sierra Nevada Healthcare System"

```